crates/tauri-bundler/src/bundle/windows/util.rs

```diff
@@ -70,8 +70,20 @@ pub fn download_webview2_offline_installer(base_path: &Path, arch: &str) -> crat
 pub fn download(url: &str) -> crate::Result<Vec<u8>> {
   log::info!(action = "Downloading"; "{}", url);
 
-  let agent = ureq::AgentBuilder::new().try_proxy_from_env(true).build();
-  let response = agent.get(url).call().map_err(Box::new)?;
+  let (agent, url) = match std::env::var("GITHUB_CDN") {
```

Comment:
I think `GITHUB_CDN` is a bit vague, maybe `TAURI_BUNDLER_TOOLS_DOWNLOAD_HOST` is better here. You also need to document that env var in `crates/tauri-cli/ENVIRONEMENT_VARIABLES.md` file

Reply:
Currently, tauri-bundler downloads files from GitHub, which is why the variable name `GITHUB_CDN` is used. 

If future versions source downloads from other websites, the variable `TAURI_BUNDLER_TOOLS_DOWNLOAD_HOST` might cause confusion or errors. To avoid this, would it be more appropriate to rename it to `TAURI_BUNDLER_TOOLS_DOWNLOAD_GITHUB_CDN` or `TAURI_BUNDLER_TOOLS_DOWNLOAD_GITHUB_TEMPLATE`?

Or is it unlikely that future versions of tauri-bundler will download files from any site other than GitHub?

Reply:
webview2, which this pr is about, isn't downloaded from github though?

Reply:
Yes, i understood this. I am not against this PR/change. I'm just saying that the GITHUB_CDN variable is used here for files that are not downloaded from github but from microsoft.

Reply:
I understand what you mean now, so I should add a source check, prioritizing CDN usage for files coming from GitHub, and for files from other source websites, downloading them based on `try_proxy_from_env`.